templates/views/privacy.md

```diff
@@ -1,62 +1,69 @@
-# Forged Alliance Forever Privacy Statement
+# Déclaration de confidentialité de Forged Alliance Forever
 
-Effective date: _May 25, 2018_
+Merci d'avoir joué à Supreme Commander : Forged Alliance dans notre communauté. Nous voulons faire en sorte qu'il soit facile pour vous de conserver vos informations personnelles et contrôler votre identité virtuelle, et dans ce but, ce document présente dans quelle mesure nous collectons ces données et de quelle façon nous les traitons
```

Comment:
Why's the effective date not mentioned?

Reply:
It got automatically deleted when importing in Poe editor, you're right. Will fix.